<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-14JT5SD" ignored="3" name="Default test(failed)" tests="1" failures="1" timestamp="21 Feb 2021 07:15:22 GMT" time="0.062" errors="0">
  <testcase name="tc02" time="0.025" classname="org.retryanalyzer.Student1">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.retryanalyzer.Student1.tc02(Student1.java:14)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- tc02 -->
  <testcase name="tc04" time="0.0" classname="org.retryanalyzer.Student1">
    <ignored/>
  </testcase> <!-- tc04 -->
  <testcase name="tc01" time="0.0" classname="org.retryanalyzer.Student1">
    <ignored/>
  </testcase> <!-- tc01 -->
  <testcase name="tc03" time="0.0" classname="org.retryanalyzer.Student1">
    <ignored/>
  </testcase> <!-- tc03 -->
</testsuite> <!-- Default test(failed) -->
